languages/elixir/exercises/concept/access/.meta/design.md

```diff
@@ -0,0 +1,44 @@
+## Learning objectives
+
+- Know of the access behavior
+- Know that the access behavior can be used on maps and keyword lists
+- Know the syntax for the access behavior
+- limitations of the access behavior
+  - doesn't work for structs
```

Comment:
Unless implemented in their defining module

Reply:
```suggestion
  - doesn't work for structs, unless the behaviour is implemented in the defining module
```